pkg/controller/jobframework/reconciler.go

```diff
@@ -398,7 +403,7 @@ func (r *JobReconciler) ReconcileGenericJob(ctx context.Context, req ctrl.Reques
 	if job.IsSuspended() {
 		// start the job if the workload has been admitted, and the job is still suspended
 		if workload.IsAdmitted(wl) {
-			log.V(2).Info("Job admitted, unsuspending")
+			log.V(5).Info("Job admitted, unsuspending")
```

Comment:
I would suggest not to change this in the PR, because level 2 is often used as default.

Reply:
Reverted.